resources/client.rb

```diff
@@ -143,6 +147,7 @@
   # end
 
   file 'delete validation.pem' do
+    sensitive new_resource.sensitive if new_resource.sensitive
```

Comment:
I don't think this is required here because of the delete action. Doesn't hurt I suppose, but I don't think the log will display the deleted file's content.